app/pulseapi/entries/admin.py

```diff
@@ -31,16 +46,27 @@ class EntryAdmin(admin.ModelAdmin):
 
     readonly_fields = (
         'id',
+        'created',
         'thumbnail_url',
         'thumbnail_image_tag',
         'bookmark_count',
     )
 
+    ordering = (
+        'created',
```

Comment:
The way I understand it this is *only* for the admin view, so not for the REST responses generated in views.py